release_notes/RELEASE.txt

```diff
@@ -38,6 +38,23 @@ New features and changes
 ========================
     Configuration:
     -------------
+    - Corrected path searched by CMake find_package command
+
+      The install path for cmake find_package files had been changed to use
+        "share/cmake"
+      for all platforms. However the trailing "hdf4" directory was not removed.
```

Comment:
Maybe: "for all platforms, but the trailing "hdf4" was left by mistake."?
Also, I wonder something like this for the first sentence:
"The install path for cmake find_package files had been changed to use "share/cmake" in version... for all platforms."  More clear, I think.  Of course, take my English with caution! :D